src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -54,12 +121,27 @@ Browse the entire dashboard catalog to find dashboard templates for all the avai
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index. Browse popular pre-built dashboards you want to install using the <Icon name="fe-chevron-left"/> and <Icon name="fe-chevron-right"/> buttons, or select the **View All** link to see the entire catalog.
 </figcaption>
 
+To install a pre-built dashboard:
+
+1. Select your account in the top right account picker.
+2. From the dashboards page, search for the pre-built dashboard you want to customize.
+3. Click the pre-built dashboard to select it. A new **dashboard setup** window should appear.
+4. Follow the instructions. If you've already instrumented the data source, you can skip this step.
+5.Click **View dashboard** to deploy the dashboard and start visualizing your data. Note that you can customize your new pre-built dashboard (see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/)). You can also change the name and settings, add new content and new widgets, and refine the charts
```

Comment:
Step 5 displays in the same line as step 4 in the build. The final period is missing ", and refine the charts."